packages/dai/src/eth/SmartContractService.js

```diff
@@ -113,24 +113,28 @@ export default class SmartContractService extends PrivateService {
 
   _getAllContractInfo() {
     let { networkName } = this.get('web3');
-    const mapping = networks.find(m => m.name === networkName);
+    let mapping = networks.find(m => m.name === networkName);
```

Comment:
This change is unnecessary—even when an object is const, its properties are still mutable